code/source/HttpClient.cpp

```diff
@@ -1,225 +1,254 @@
 #include "stdafx.h"
 #include "HttpClient.h"
-#include "PlayFabSettings.h"
 #include "JsonUtils.h"
+#include "SdkVersion.h"
 
 namespace PlayFab
 {
+    constexpr char productionEnvironmentURL[]{ ".playfabapi.com" };

```

Comment:
currently this is not configurable by titles. We could expose a way to set this if needed though.